assets/src/blocks/Media/MediaEmbedPreview.js

```diff
@@ -0,0 +1,3 @@
+export const MediaEmbedPreview = (props) => {
```

Comment:
Not a big deal, but this could be a tiny bit cleaner by removing the `return` statement, a.k.a 
```
export const MediaEmbedPreview = props => <div dangerouslySetInnerHTML={{ __html: props.html }} />;
```
(same for `MediaElementVideo` above actually 🙂)